src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/AppearancePreferencesPane.R

```diff
@@ -15,7 +18,7 @@ fivenum <- function(x) {
   i <- c(n5, n4, n3, n2, n1)
   
   # compute quartile values
-  x <- sort(x)
+  x <- sort(x, decreasing = FALSE)
```

Comment:
```suggestion
  x <- sort(x)
```
superflou. actually NA highlight like TRUE FALSE